### PR TITLE
Performance improvement for initial mount.

### PR DESCRIPTION
**Pull request by tokafish**
First off, thanks for all your work on this library!

I've been trying to optimize the initial mount of a react app, and noticed a few spots where I thought an improvement to react-virtualized might help.

I'll follow up this PR with all of the items in the checklist, but I wanted to get your thoughts on what you thought of this change. Thanks!

### CODE REVIEW

**Comment by tokafish** [comment]
@wuweiweiwu updated, sorry about that!

**Comment by tokafish** [comment]
@wuweiweiwu Hey - just wanted to check in on this - thanks!

**Comment by jincan39** [comment]
https://github.com/bvaughn/react-virtualized/issues/1290 any thinking on this dude? thanks @wuweiweiwu

**Comment by tokafish** [comment]
@wuweiweiwu Just checking in again - thanks!

**Comment by wuweiweiwu** [comment]
Will take a look soon!

File: source/Grid/Grid.js [hunk 1]
```diff
@@ -619,8 +619,8 @@ class Grid extends React.PureComponent<Props, State> {
     }
 
     if (
-      (typeof scrollLeft === 'number' && scrollLeft >= 0) ||
-      (typeof scrollTop === 'number' && scrollTop >= 0)
+      (typeof scrollLeft === 'number' && scrollLeft > 0) ||
```
Comment (wuweiweiwu) [inline source/Grid/Grid.js]: why remove 0?

File: source/Grid/Grid.js [hunk 2]
```diff
@@ -637,10 +637,10 @@ class Grid extends React.PureComponent<Props, State> {
     if (this._scrollingContainer) {
       // setting the ref's scrollLeft and scrollTop.
       // Somehow in MultiGrid the main grid doesn't trigger a update on mount.
-      if (this._scrollingContainer.scrollLeft !== this.state.scrollLeft) {
```
Comment (wuweiweiwu) [inline source/Grid/Grid.js]: why do we check if the scrollLeft is 0 again here?

**Pull request closed without merging**